pkg/render/node.go

```diff
@@ -344,7 +397,7 @@ func (c *nodeComponent) nodeRole() *rbacv1.ClusterRole {
 				// Used for creating service account tokens to be used by the CNI plugin.
 				APIGroups:     []string{""},
 				Resources:     []string{"serviceaccounts/token"},
-				ResourceNames: []string{"calico-node"},
+				ResourceNames: []string{"calico-node", "calico-cni-plugin"},
```

Comment:
We want to remove the ability for calico-node to create serviceaccount tokens for itself, so I think you should remove `calico-node` from this list

Reply:
This comment still needs addressing @MichalFupso - need to remove "calico-node" from ResouceNames.